### PR TITLE
ti_misp - Fix duplicate requests for page 1

### PR DESCRIPTION
**Pull request by andrewkroh**
Avoid requesting page 1 more than once. The page counter within httpjson begins at 0 and the page parameter within MISP starts at 1.

The results were verified using this against the request tracer log from the system test.

    jq -c '{"http.request.body.content":.["http.request.body.content"], "transaction.id":.["transaction.id"]}' tracer.ndjson | grep -v null
    {"http.request.body.content":"{\"limit\":\"10\",\"page\":\"1\",\"returnFormat\":\"json\",\"timestamp\":\"1686065582\"}","transaction.id":"L1FFIRQ23HJ1E-1"}
    {"http.request.body.content":"{\"limit\":\"10\",\"page\":\"2\",\"returnFormat\":\"json\",\"timestamp\":\"1686065582\"}","transaction.id":"L1FFIRQ23HJ1E-2"}
    {"http.request.body.content":"{\"limit\":\"10\",\"page\":\"3\",\"returnFormat\":\"json\",\"timestamp\":\"1686065584\"}","transaction.id":"L1FFIRQ23HJ1E-3"}

Relates #6479

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/ti_misp/_dev/deploy/docker/files/config.yml [hunk 1]
```diff
@@ -9,70 +9,77 @@ rules:
       - status_code: 200
         body: |-
           {
-            "response": [{
```
Comment (andrewkroh) [inline packages/ti_misp/_dev/deploy/docker/files/config.yml:12]: NOTE: I formatted this JSON body and then split the existing events across page 1 and page 2.

File: packages/ti_misp/data_stream/threat/_dev/test/system/test-default-config.yml [hunk 2]
```diff
@@ -9,3 +9,6 @@ data_stream:
     interval: 10m
     initial_interval: 10m
     enable_request_tracer: true
+
+assert:
+  hit_count: 3
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat/_dev/test/system/test-default-config.yml:14]: Final new line.

File: packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs [hunk 3]
```diff
@@ -56,7 +56,7 @@ response.request_body_on_pagination: true
 response.pagination:
 - set:
     target: body.page
-    value: '[[if (ne (len .last_response.body.response) 0)]][[add .last_response.page 1]][[end]]'
+    value: '[[if (ne (len .last_response.body.response) 0)]][[add .last_response.page 2]][[end]]'
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs:59]: This is surprising enough to warrant a comment.

File: packages/ti_misp/data_stream/threat_attributes/_dev/test/system/test-default-config.yml [hunk 4]
```diff
@@ -9,3 +9,6 @@ data_stream:
     interval: 10m
     initial_interval: 10m
     enable_request_tracer: true
+
+assert:
+  hit_count: 5
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/_dev/test/system/test-default-config.yml:14]: Final new line.

File: packages/ti_misp/data_stream/threat_attributes/agent/stream/httpjson.yml.hbs [hunk 5]
```diff
@@ -48,7 +48,7 @@ response.request_body_on_pagination: true
 response.pagination:
 - set:
     target: body.page
-    value: '[[if (ne (len .last_response.body.response) 0)]][[add .last_response.page 1]][[end]]'
+    value: '[[if (ne (len .last_response.body.response) 0)]][[add .last_response.page 2]][[end]]'
```
Comment (efd6) [inline packages/ti_misp/data_stream/threat_attributes/agent/stream/httpjson.yml.hbs:51]: And here too.

File: packages/ti_misp/_dev/deploy/docker/files/config.yml [hunk 6]
```diff
@@ -9,70 +9,77 @@ rules:
...
-                  "Galaxy": [],
-                  "Object": [{
-                    "Attribute": [{
+                  "Attribute": [
+                    {
                       "Galaxy": [],
                       "ShadowAttribute": [],
                       "category": "Payload delivery",
-                      "comment": "",
+                      "comment": "filename contect for test event 3",
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply packages/ti_misp/_dev/deploy/docker/files/config.yml:20]: That diff is a bit misleading in that I did not add this string. Viewing without whitespace is better for this file.

But my guess would be "content". 🤷

File: packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs (hunk 3 above)
Comment (andrewkroh) [reply packages/ti_misp/data_stream/threat/agent/stream/httpjson.yml.hbs:60]: Good call. Done.

File: packages/ti_misp/_dev/deploy/docker/files/config.yml (hunk 6 above)
[the remark this answers is not in the document]
Comment (andrewkroh) [reply packages/ti_misp/_dev/deploy/docker/files/config.yml]: Changed to "content".

**Review by efd6 — APPROVED** [review]

**Pull request merged**